### ISSUE TITLE
make's command arguments like VERBOSE need to be passed to subprojects

### ISSUE THREAD

**Issue by evaleev** [ValeevGroup/tiledarray#55]

**Comment by justusc** [comment]
I tried to make this work but the verbosity for auto tools is hard coded at configure time. And the Makefiles generated by auto tools does not respect the `VERBOSE` variable.

**Comment by evaleev** [comment]
The real issue is that we are still using autotools for MADness ... I'm trying to switch to cmake-based configure for MADNESS when compiled from within TA.

**Comment by justusc** [comment]
That is great. The process is relatively simple. Look at the elemental configure file o set up in madness for an example. https://github.com/m-a-d-n-e-s-s/madness/blob/master/src/madness/external/CMakeLists.txt

**Comment by evaleev** [comment]
what makes most sense is to move all duplicate checks to MADNESS (e.g. MPI), with only supplemental checks done in TA + checks unique to TA (e.g. Eigen). Elemental compilation is also moving to MADNESS, with TA-specific overrides to be implemented.

**Comment by justusc** [comment]
That seems reasonable, but it means MADNESS cannot be built separately. I'm OK with this approach since it simplifies a lot of things.

Are you thinking of making MADNESS a subtree or submodule?

**Comment by evaleev** [comment]
I don't see why we can't build MADNESS separately this way, we just need to export everything through cmake configure file; only if additional checks are needed we'll do them in TA.

As a first step let's get this working with MADNESS being a cmake ExternalProject ... this will already do much work in the way of resolving any differences in variable names etc. This will make turning it into a subtree in TA trivial.

**Comment by justusc** [comment]
I do not want to use ExternalProject again. I moved away from this because:
1. It will prefer previously installed version of the external project, even when the project has been updated.
2. It is far too aggressive about downloading the external project. It forces you to pull the repository even when it is already downloaded. (I also don't want to include tar balls to work around this issue either; the current code works well at pulling MADNESS at the right time).
3. Configuring the external project was deferred to build time. This often lead to confusion and frustration. I would much rather configure at build time.
